package.json

```diff
@@ -1,82 +1,35 @@
-        "url": "git+https://github.com/near/near-api-js.git"
-    },
-    "homepage": "https://github.com/near/near-api-js",
-    "main": "lib/index.js",
-    "browser": "lib/browser-index.js",
-    "types": "lib/index.d.ts",
-    "dependencies": {
-        "bn.js": "5.2.1",
-        "borsh": "^0.7.0",
-        "bs58": "^4.0.0",
-        "depd": "^2.0.0",
-        "error-polyfill": "^0.1.3",
-        "http-errors": "^1.7.2",
-        "js-sha256": "^0.9.0",
-        "mustache": "^4.0.0",
-        "node-fetch": "^2.6.1",
-        "text-encoding-utf-8": "^1.0.2",
-        "tweetnacl": "^1.0.1"
-    },
-    "devDependencies": {
-        "@types/bn.js": "^5.1.0",
-        "@types/http-errors": "^1.6.1",
-        "@typescript-eslint/eslint-plugin": "^2.18.0",
-        "@typescript-eslint/parser": "^2.18.0",
-        "browserify": "^16.2.3",
-        "bundlewatch": "^0.3.1",
-        "danger": "^11.1.1",
-        "danger-plugin-yarn": "^1.3.2",
-        "eslint": "^6.5.1",
-        "husky": "^7.0.4",
-        "in-publish": "^2.0.0",
-        "jest": "^26.0.1",
-        "localstorage-memory": "^1.0.3",
-        "near-hello": "^0.5.1",
-        "rimraf": "^3.0.0",
-        "semver": "^7.1.1",
-        "ts-jest": "^26.5.6",
-        "ts-morph": "^15.1.0",
-        "typedoc": "^0.20.36",
-        "typedoc-neo-theme": "^1.1.1",
-        "typescript": "^3.5.1",
-        "uglifyify": "^5.0.1"
-    },
-    "keywords": [],
-    "license": "(MIT AND Apache-2.0)",
-    "scripts": {
-        "dist": "yarn browserify && yarn doc",
-        "browserify": "browserify browser-exports.js -i node-fetch -i http -i https -o dist/near-api-js.js && browserify browser-exports.js -i node-fetch -g uglifyify -o dist/near-api-js.min.js",
-        "prebrowserify": "yarn build",
-        "prepublish": "not-in-install && (yarn build && yarn browserify) || in-install",
-        "compile": "tsc -p ./tsconfig.json",
-        "dev": "yarn compile -w",
-        "doc": "typedoc src && touch docs/.nojekyll",
-        "build": "yarn compile",
-        "test": "jest test",
-        "lint": "eslint test && eslint src/**/*.ts",
-        "fix": "eslint test --fix && eslint src/**/*.ts --fix",
-        "prefuzz": "yarn build",
-        "fuzz": "jsfuzz test/fuzz/borsh-roundtrip.js test/fuzz/corpus/",
-        "prepare": "husky install"
-    },
-    "bundlewatch": {
-        "files": [
-            {
-                "path": "dist/near-api-js.min.js",
-                "maxSize": "105kB"
-            }
-        ]
-    },
-    "files": [
-        "lib",
-        "dist",
-        "browser-exports.js"
+  "name": "@near-api-js/monorepo",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "packages/*"
     ],
-    "author": "NEAR Inc"
+    "nohoist": [
+      "**/near-hello",
+      "**/near-hello/**"
+    ]
+  },
+  "scripts": {
+    "build": "yarn lerna run build",
+    "clean": "yarn lerna run clean --concurrency 4",
+    "lint": "yarn lerna run lint --stream --concurrency 1",
```

Comment:
Why do we need to enforce any concurrency limit at all? Why not just use the default `10` for 🚀 🚀

Reply:
Yeah limits should be removed for lint and clean. I'm adding to test though for cleaner output